Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -65,6 +65,7 @@ public class WebTreeView : WebControl, IWebTreeView, IPostBackEventHandler, IRes
     private const string c_nodeIconTMinus = "sprite.svg#TreeViewTMinus";
     private const string c_nodeIconTPlus = "sprite.svg#TreeViewTPlus";
     private const string c_nodeIconWhite = "sprite.svg#TreeViewWhite";
+    private const string c_treeViewSeparator = "treeViewSeparator";
```

Comment:
This should probably be implemented like the other css classes -> virtual property see `CssClassNodeChildren` for an example